js/components/src/components/chat/chat-box.tsx

```diff
@@ -74,6 +74,14 @@ export function ChatBox({
 
   let agent = usePDSAgent();
 
+  // if no agent, then show a spinner
+
+  if (!agent?.did) {
+    <View style={[layout.flex.row, layout.flex.alignCenter, gap.all[1]]}>
+      <Loader size={16} />
+    </View>;
+  }
```

Comment:
Haha looks like these lines never did anything, no `return` -- the current message is here https://github.com/streamplace/streamplace/blob/10ecea7b28b19b72cab54b3d15acea663e3c0e93/js/app/components/mobile/chat.tsx#L160